### PR TITLE
LLT-5252: Clean unneeded words

### PR DESCRIPTION
**Pull request by lcruz99**
Gitlab URL's updated so these variables have to be updated.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: .github/workflows/gitlab.yml [hunk 1]
```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v1.1.12
+          triggered-ref: LLT-5252_update_gitlab_url
```
Comment (stalowyjez) [inline .github/workflows/gitlab.yml:34]: I guess you will update it to `v1.1.13` after merging?

File: .unreleased/LLT-5252 [hunk 2]
```diff
@@ -0,0 +1 @@
+ci: Update gitlab urls
```
Comment (stalowyjez) [inline .unreleased/LLT-5252:1]: I guess when it is not a PR introducing sth used by apps this file should be empty

**Review by stalowyjez — COMMENTED** [review]
I find the name of this PR a bit misleading - I think sth like "fix moose path" would be nicer

File: .github/workflows/gitlab.yml (hunk 1 above)
Comment (lcruz99) [reply .github/workflows/gitlab.yml:34]: correct, thanks for leaving a reminder :)

File: .unreleased/LLT-5252 (hunk 2 above)
Comment (lcruz99) [reply .unreleased/LLT-5252:1]: How can I tell that to `check-changelog` job? 👀

**Pull request closed without merging**